rune/libenclave/intelsgx/preload/device_linux.go

```diff
@@ -24,12 +24,13 @@ func loadLibrary(p string) {
 // Due to the design of runelet, the Enclave Runtime PAL is loaded
 // in host but launched in container. The fact that certain libraries
 // from Intel SGX PSW would use dlopen() to further load
-// libsgx_launch.so, which means the container has to have it. In
-// order to ensure all libraries dependent by Enclave Runtime PAL
-// are completely loaded in host, preload them prior to switch
+// libsgx_launch.so and libsgx_epid.so, which means the container has 
```

Comment:
It is not necessary to provide the details about the lib names, because it is predictable that more and more libs will need to be preloaded. Plz change to:
```
use dlopen() to further load supporting libraries, which means the container has to contain them.
```